docs/src/components/Layout/SecondaryNav.tsx

```diff
@@ -185,12 +185,12 @@ export const SecondaryNav = (props) => {
 
 export const Sidebar = () => {
   return (
-    <aside className="docs-sidebar">
```

Comment:
We also have two asides; the other is the table of contents for the page itself. Since this sidebar element only houses the navigation, I changed this element to a <nav> (with aria label) with nested divs inside of it since the nav is the only semantic element needed.